libs/constants/index.js

```diff
@@ -153,6 +153,38 @@ export const cspToString = (csp) =>
 
 export const CSP_VALUE = cspToString(CSP_DIRECTIVES);
 
+// Always update client/src/setupProxy.js when adding/removing extensions, or it won't work on the dev server!
+export const AUDIO_EXT = ["mp3", "ogg"];
+export const FONT_EXT = ["woff2"];
```

Comment:
We allow `font/woff` as mime type but not the file extension. Can we make this consistent?

Reply:
Fixed in 7e3326aab.